src/libraries/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -158,6 +160,7 @@
     <Reference Include="System.ComponentModel.TypeConverter" />
     <Reference Include="System.Diagnostics.Debug" />
     <Reference Include="System.IO.FileSystem" />
+    <Reference Include="System.IO.FileSystem.AccessControl" Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))" />
```

Comment:
Instead of adding this verbose condition to each item, can we group them in a shared itemgroup where the condition is set to the whole itemgroup?

Reply:
These will go away in 4-5 months when we target NET7 so I don't care much about the added verbosity. Let's keep it as is meanwhile.